content/concepts/introduction/users.md

```diff
@@ -0,0 +1,60 @@
+| **Ceramic** | Ceramic's use of libp2p has contributed to its ability to handle high-volume application data with web-scale volume and latency; libp2p has also enabled IDX, Ceramic's decentralized identity solution. |
+| **Nethermind, juno, dotnet-libp2p** | By leveraging libp2p, Nethermind has positioned its projects (Juno and dotnet-libp2p) to contribute significantly to the decentralization and scalability of blockchain networks, particularly in the Ethereum and Starknet ecosystems. |
+| **Drand / Randamu** | Libp2p enables direct peer-to-peer communication between Drand nodes, allowing for decentralized randomness generation and distribution. |
+| **Avail** | By utilizing libp2p, Avail has created a robust, scalable, and efficient light client network that enhances its data availability guarantees and improves user experience in interacting with the blockchain. |
+| **Prysm (Prysmatic Labs)** | The adoption of libp2p has allowed Prysm to be interoperable with other Ethereum consensus clients, contributing to client diversity in the ecosystem. |
+| **Lighthouse (Sigma Prime)** | The adoption of libp2p aligns with Lighthouse's security-first mindset, allowing for extensive reviews and monitoring of the networking stack. |
+| **Arbitrum** | Arbitrum, which uses libp2p as part of its networking stack, has established itself as the most dominant and adopted L2 chain in the Ethereum rollup ecosystem, commanding approximately 50% of the L2 market share. |
+| **Spacedrive** | Spacedrive implements a lazy connection system using libp2p, which establishes connections only when needed and closes them after a period of inactivity to preserve battery life and reduce network usage. |
+| **Nym** | Nym's mixnet technology is integrated with libp2p to provide network-level privacy protection. The Nym libp2p module can be used by Ethereum validators to shield their network traffic, preventing IP address and metadata leakage. |
+| **Mantle** | Mantle's use of libp2p contributes to its efficient peer-to-peer communication and block propagation, which are crucial for its L2 solution. |
+| **Nillion** | Compute nodes in the Nillion network are configured as relay servers using libp2p's Circuit Relay protocol. |
+| **Aave** | Leveraging libp2p's flexibility, Aave has successfully expanded to multiple blockchain networks, including Ethereum, Polygon, and Avalanche - increasing Aave's ecosystem expansion, with over 100 integrated projects and applications. |
+| **Bittorrent FS** | By leveraging libp2p for peer-to-peer networking and stream management, BTFS has improved its scalability, supporting a growing ecosystem of users and developers. |
+| **Internet Computer (IC)** | Internet Computer's networking layer built on libp2p has successfully implemented Byzantine Fault Tolerant (BFT) protocols, enhancing the security and reliability of the IC. |
```

Comment:
Does this use libp2p? I was having trouble finding reliable information here which makes me think not.